packages/ssi-sdk-core/src/utils/database.ts

```diff
@@ -2,19 +2,18 @@ export const flattenArray = <T>(args: { items: Array<T | Array<T>> }): Array<T>
 
 export const flattenMigrations = <T>(args: { migrations: Array<T | Array<T>> }): Array<T> => args.migrations.flat() as Array<T>
 
-/**
- * It should accept queryRunner from the typeorm
```

Comment:
Okay an now add the remark back to the type please, so people know you are creating a type that mimics TypeOrm